p2p-specs/p2p-interface.md

```diff
@@ -494,20 +495,30 @@ Request Content:
 
 ```
 (
-  mempool: Bytes32
   offset: uint64
 )
 ```
 
 Response Content: 
 ```
 (
-  more_flag: uint64
+  total_count: uint64
```

Comment:
We did discuss this in one of our initial meetings and chose to go ahead with `more_flag` instead of `total_count` because the count could change in-between two subsequent requests.